README.md

```diff
@@ -1 +1,62 @@
-# client-examples
\ No newline at end of file
+# Description of client-examples
+This repository contains example of Kafka clients. 
+Both producer and consumer clients are transformed into docker images which allows to deploy them on OpenShift platform.
```

Comment:
"transformed" doesn't sound right: "assembled into"

...which allows them to be deployed on OpenShift